Added documentation for running PL from WSL2 image. (#3258)
PrairieLearn/PrairieLearn: docs/installing.md

```sh
docker run -it --rm -p 3000:3000 [other args] prairielearn/prairielearn:1.2.3
```

## Running PrairieLearn from a WSL2 instance

If you are using Windows with WSL2, you should be able to run Docker from another WSL2 instance. In order to that, you need to follow these instructions:

> * Open the Docker Dashboard, and click on Settings (the gear button at the top of the interface). Under General, ensure the "Use the WSL2 based engine" option is selected. Then, under Resources, select "WSL integration", and enable the option "Enable integration with my default WSL distro". Also enable integration with the distros listed below that you want to access docker from. Click on "Apply & Restart" for the settings to apply.

Reviewer: ```suggestion
* Open the Docker Dashboard, and click on Settings (the gear button at the top of the interface).
    * Under General, ensure the "Use the WSL2 based engine" option is selected.
    * Then, under Resources, select "WSL integration", and enable the option "Enable integration with my default WSL distro".
    * Also enable integration with any listed distros that you want to access docker from.
    * Click on "Apply & Restart" for the settings to apply.
```
bullets for easier scanning

* On the shell of your WSL2 instance, make sure the instance has the `docker` command installed. The installation process may depend on your distribution, but most distributions provide a `docker` package.

> * Now you should be able to start PrairieLearn with the following command (assuming your course is stored under `/mnt/c/Users/yourname/git/pl-tam212`:

Reviewer: ```suggestion
* Now you should be able to start PrairieLearn with the following command (assuming your course is stored under `/mnt/c/Users/yourname/git/pl-tam212`):
```

```sh
docker run -it --rm -p 3000:3000 -v /mnt/c/Users/yourname/git/pl-tam212:/course prairielearn/prairielearn
```

> If you plan on running externally graded questions or workspaces in local development, please see [this section](../externalGrading/#running-locally-on-docker) for a slightly different docker launch command.

Reviewer: ```suggestion
If you plan on running externally graded questions or workspaces in local development, please see the [docker section in the external grading docs](../externalGrading/#running-locally-on-docker) for a slightly different docker launch command.
```
accessibility nit -- informative link text
